examples/cloud-backup-snapshot-export-bucket/privateEndpoint.json

```diff
@@ -0,0 +1,41 @@
+{
+  "AWSTemplateFormatVersion" : "2010-09-09",
```

Comment:
this filename says privateEndpoint.json should we update this for cloud-backup-snapshot-export-bucket.json example instead?